pages/en/blog/announcements/diving-into-the-nodejs-website-redesign.md

```diff
@@ -0,0 +1,118 @@
+
+[Next.js](https://nextjs.org/) was a natural progression, notable for its flexibility and power. For instance, the site is still statically built for end-user speed and foundational hosting independence but leverages Next.js's incremental static regeneration to pick up dynamic content like releases. React offered an authoring experience not only more aligned with the current skillsets of contributors, but also with a larger ecosystem of tools at our disposal.
+
+We had a close partnership with [Vercel](https://vercel.com/) along this journey. They provided direct support when the scale of the site strained webpack's memory management during static export. This was a symbiotic relationship of sorts. Our needs pushed their platform to improve, and their platform enabled us to build a better site. We beta-tested new releases before public availability, a real-world stress test for the framework.
+
+In April 2023, we performed a miniature cutover. This is a bit of an ironic statement. The [pull request](https://github.com/nodejs/nodejs.org/pull/4991) was 1600 files and pushed the GitHub UI to the limits of its rendering capabilities. The site's infrastructure would change, but the look and feel, content, and authoring experience would remain unchanged. This was a critical milestone—proof we could rebuild the airplane in flight.
+
+## Redesign
+
+Also in 2023, the [OpenJS Foundation](https://openjsf.org/) began a redesign of their site. They hired [Hayden Bleasel](https://haydenbleasel.com/) and were kind enough to include nodejs.org in the scope of his contract. Hayden brought a modern design to the site and helped us think through the multi-faceted use cases we encountered. The result was a [Figma](https://www.figma.com/) document that we could use to guide our development. Included were UX flows, dark/light modes, page layouts, mobile viewport considerations, and a component breakdown.
+
+![A screenshot of the nodejs.org website redesign in Figma.](/static/images/blog/announcements/2024-nodejs-figma.png)
+
+Realizing the design as code was next, and an effort distributed across the world. A lot of emphasis was placed on sequential build-up of foundational design elements and structured component hierarchies. We built variants of components from day one and considered internationalization at the outset. This reduced rework and made any task accessible to a newcomer. We started building components in isolation via Storybook and [Chromatic](https://www.chromatic.com/)'s hosted instance. We choose to use [Tailwind CSS](https://tailwindcss.com/), but with an emphasis on design tokens and applied CSS. This common language helped newcomers orient to our approach and translate the Figma.
```

Comment:
It might be worth mentioning why we chose from the beginning to use Storybook to create an environment where we could easily experiment/prototype, iterate, and validate components. And how Chromatic allowed us to QA these components.

Reply:
done via https://github.com/nodejs/nodejs.org/pull/6524/commits/4751a1383d8ab340a7d90c681fe852983f94689b